cmd/spec/parameters.go

```diff
@@ -26,3 +26,12 @@ var OffsetQueryParam = Parameter{
 	Required:    false,
 	In:          "query",
 }
+
+var TokenQueryParam = Parameter{
+	Name:        "token",
+	Description: "The token used for requesting the next page of results; empty for the first page",
```

Comment:
```suggestion
	Description: "The token used for requesting the next page of results; empty for the last page",
```

Reply:
That was not what I meant here, I meant that an empty token leads to the first page.

Reply:
Oh it is a bit unclear but it is fine, thanks.